.github/workflows/fuzzers.yml

```diff
@@ -2,10 +2,10 @@ name: Build fuzzers
 on:
   pull_request:
     paths:
-      - 'docker/**'  # Base images changes.
+      - 'docker/**'  # Base images image changes.
       - 'fuzzers/**' # Changes to fuzzers themselves.
       - 'benchmarks/**'  # Changes to benchmarks.
-      - 'src_analysis/**' # Changes that affect what gets built.
+      - 'src_analysis/**' # Changes that effect what gets built.
```

Comment:
affect seems right ?

Reply:
Yup, don't know how I've somehow unlearned the difference lately.